### ISSUE TITLE
Import Widgets after the after_content_import_execution hook.

### ISSUE THREAD

**Issue by shofer4eto** [proteusthemes/one-click-demo-import#161]
Hi,

Thanks for the great plugin. I implemented Option tree options import and have it hooked to the after_content_import_execution action. I want the Widgets to import after that custom import as I have several dynamic sidebars that I get set with the Option tree options import and I want to populate them with widgets only after they are created... How do I do that?

**Comment by capuderg** [comment]
Hi,

if you look at this file: https://github.com/proteusthemes/one-click-demo-import/blob/master/inc/ImportActions.php

You will see that the widgets import and the Redux import are hooked into `pt-ocdi/after_content_import_execution`. Why not just use this hook to import the option tree as well? To run it before "widgets import" use a lower priority number (-> lower then 20).

Take care!

**Comment by shofer4eto** [comment]
Hi,

Thanks for the quick reply! 
Exactly what I was doing, seemed like this was not the reason, widgets did get imported after the Theme options import, but somehow the dynamic sidebar area (created via theme options) doesn't get found, here's what the log says:

---Importing widgets---
contact_sidebar : Sidebar does not exist in theme (moving widget to Inactive)

I will dig further in this, thanks!

**Comment by minimaluminium** [comment]
I'm also having similar issue as well. There are several custom widget areas created by WooSidebars (https://wordpress.org/plugins/woosidebars/) and the widgets inside these widget areas are moved into "Inactive Widgets" after the import. It seems like that these widget areas were imported after widget imports. Any possible solution on this?

**Comment by capuderg** [comment]
Hi @minimaluminium,

thank you for sharing your solution.
